<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statusbar</class>
 <widget class="QWidget" name="Statusbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>29</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QComboBox" name="unit"/>
    </item>
    <item>
     <widget class="QComboBox" name="imagePreviewQuality">
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScrSpinBox" name="zoom">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3200</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomOut">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../../resources/icons/16/zoom-out.png</normalon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomDefault">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../../resources/icons/16/zoom-original.png</normalon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomIn">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../../resources/icons/16/zoom-in.png</normalon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ScrSpinBox" name="page">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3200</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomOut_2">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../../resources/icons/16/go-first.png</normalon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomOut_3">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../../resources/icons/16/go-previous.png</normalon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomOut_4">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../../resources/icons/16/go-next.png</normalon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="zoomOut_5">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>../../resources/icons/16/go-last.png</normalon>
       </iconset>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
